infinex-app-registry-test/testnets/infinex-app-registry-test.ethereum-sepolia.toml

```diff
@@ -0,0 +1,24 @@
+#  pnpm cannon build infinex-app-registry-test/testnets/infinex-app-registry-test.ethereum-sepolia.toml --chain-id 11155111 --dry-run --upgrade-from
+#  pnpm cannon publish infinex-app-registry-test:3.0.0@app-registry --chain-id 11155111 --exclude-cloned
+
+name = "infinex-app-registry-test"
+version = "3.0.0"
+preset = "app-registry"
+description = "Approved apps for the Infinex App Registry"
+
+[pull.InfinexCoreTest]
+source = "infinex-core-test:latest@core"
```

Comment:
again, I feel like this should specify a version, else a version that's not ready will get added